pages/help/[page].tsx

```diff
@@ -0,0 +1,114 @@
+import addSiteLocation from '../../content/help/addSiteLocation.md';
+import { withSites } from '~/lib/sites';
+import ReactMarkdown from 'react-markdown';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { LinkProps } from 'next/link';
+
+export const pages: any = {
```

Comment:
Does this cause type errors if the any is removed?
```suggestion
export const pages = {
```